Assets/MRTK/Providers/Oculus/XRSDK/Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.asmdef

```diff
@@ -2,9 +2,11 @@
     "name": "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus",
     "references": [
         "Microsoft.MixedReality.Toolkit",
-        "Microsoft.MixedReality.Toolkit.Providers.XRSDK"
+        "Microsoft.MixedReality.Toolkit.Providers.XRSDK",
+        "Microsoft.MixedReality.Toolkit.XRSDK.Oculus.Handtracking.Utilities",
+        "Oculus.VR",
+        "Unity.XR.Oculus"
     ],
-    "optionalUnityReferences": [],
     "includePlatforms": [
         "Android",
```

Comment:
just checking. are we only supporting quest with this provider? or are we also supporting rift?

Reply:
I think it works on quest and rift based on the online documentation for the integration package and the xrsdk pipeline, but not completely sure how some components like handtracking work on the rift.